macro/src/lib.rs

```diff
@@ -3,8 +3,7 @@
 // see LICENSE for license details.
 
 //! Subxt macro for generating Substrate runtime interfaces.
-
-extern crate proc_macro;
+#![allow(clippy::manual_unwrap_or_default, missing_docs)]
```

Comment:
is this `allow` needed?

Reply:
Yeah, for some reason workspace lint is not getting propagated here for manual unwrap